test/cpp/api/modules.cpp

```diff
@@ -239,7 +239,7 @@ TEST_CASE("modules_cuda", "[cuda]") {
   SECTION("1") {
     Linear model(5, 2);
     model->cuda();
-    auto x = torch::randn({10, 5}, at::device(at::kCUDA).requires_grad());
+    auto x = torch::randn({10, 5}, at::device(at::kCUDA).requires_grad(true));
```

Comment:
What happened here?

Reply:
I removed the = true default from the requires_grad getter since it was actually conflicting with the requires_grad getter (both could take 0 arguments)